packages/viewport/package.json

```diff
@@ -0,0 +1,34 @@
+{
+	"name": "@wordpress/viewport",
+	"version": "1.0.0-alpha.3",
```

Comment:
I guess it's not that important but I thought we were using -beta.0 for the first unpublished versions. Since we publish stable versions directly, it's not very important.

Reply:
I think we are going to publish `1.0.0` later this week for all packages to make publishing with 2FA maintinanable 😄